From 5908bf57faaec1b129f2a41daa28995571d5a444 Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Thu, 19 Jan 2017 11:48:22 +0800
Subject: [PATCH] ARM: rockchip_linux_deconfig: make MIDGARD CONFIG back

Since commit "gpu: mali: allow midgard for linux build as a module",
we have move midgard config define from default.mk to defconfig.

Change-Id: Ia24f6bf8489404d52954ee72a83e66a168d97fc4
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index 467a67f5e1e9..9c0d63ea1354 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -285,6 +285,11 @@ CONFIG_DRM_UDL=y
 CONFIG_DRM_PANEL_SIMPLE=y
 CONFIG_DRM_DW_HDMI_I2S_AUDIO=y
 CONFIG_MALI_MIDGARD_FOR_LINUX=y
+CONFIG_MALI_MIDGARD=y
+CONFIG_MALI_DEVFREQ=y
+CONFIG_MALI_EXPERT=y
+CONFIG_MALI_PLATFORM_THIRDPARTY=y
+CONFIG_MALI_PLATFORM_THIRDPARTY_NAME="rk"
 CONFIG_MALI400=y
 # CONFIG_MALI400_PROFILING is not set
 # CONFIG_MALI_DVFS is not set
-- 
2.35.3

